qt/qt_common/map_widget.cpp

```diff
@@ -61,9 +67,17 @@ void MapWidget::BindHotkeys(QWidget & parent)
       {Qt::Key_Equal, SLOT(ScalePlus())},
       {Qt::Key_Plus, SLOT(ScalePlus())},
       {Qt::Key_Minus, SLOT(ScaleMinus())},
+      {Qt::Key_Right, SLOT(MoveRight())},
+      {Qt::Key_Left, SLOT(MoveLeft())},
+      {Qt::Key_Up, SLOT(MoveUp())},
+      {Qt::Key_Down, SLOT(MoveDown())},
       {static_cast<int>(Qt::ALT) + static_cast<int>(Qt::Key_Equal), SLOT(ScalePlusLight())},
       {static_cast<int>(Qt::ALT) + static_cast<int>(Qt::Key_Plus), SLOT(ScalePlusLight())},
       {static_cast<int>(Qt::ALT) + static_cast<int>(Qt::Key_Minus), SLOT(ScaleMinusLight())},
+      {static_cast<int>(Qt::ALT) + static_cast<int>(Qt::Key_Right), SLOT(MoveRightSmooth())},
+      {static_cast<int>(Qt::ALT) + static_cast<int>(Qt::Key_Left), SLOT(MoveLeftSmooth())},
+      {static_cast<int>(Qt::ALT) + static_cast<int>(Qt::Key_Up), SLOT(MoveUpSmooth())},
+      {static_cast<int>(Qt::ALT) + static_cast<int>(Qt::Key_Down), SLOT(MoveDownSmooth())},
```

Comment:
According to this [QT's documentation](https://doc.qt.io/qt-6/qkeysequence.html), 
```cpp
QKeySequence(tr("Ctrl+X, Ctrl+C"));
QKeySequence(Qt::CTRL | Qt::Key_X, Qt::CTRL | Qt::Key_C);
QKeySequence(Qt::CTRL + Qt::Key_X, Qt::CTRL + Qt::Key_C); // deprecated
```
You should use `|` instead of `+` for `QKeySequence`. Also, using `QKeySequence` for `Hotkey::m_key` might help reduce the number of conversions to/from `int` (`static_cast<int>`)

Reply:
Indeed, good point. I was following what it was done before, but now I have used QKeySequence as the Hotkey and removed the static_cast conversions.